gramine/README.md

```diff
@@ -0,0 +1,75 @@
+## How to run demo with Gramine
+
+1. Start a SPS container:
+```shell
+docker run -d -v /opt/multiple/sessions:/graphene/workplace/sessions -v /opt/secret-prov/certs/:/graphene/workplace/certs -p 8080:8080 -p 4433:4433 -e SPS_USERNAME=admin -e SPS_PASSWORD=admin --name iexec-sps iexechub/iexec-sps:bf3bb00-dev
```

Comment:
Use a better image tag for SPS ?

Reply:
Sure, see https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/54/commits/443a67615c9d309fa7e2771dc0b85dcedab6092b.